doc/sphinxext/numpy_ext/docscrape_sphinx.py

```diff
@@ -78,13 +84,16 @@ def _str_member_list(self, name):
                     others.append((param, param_type, desc))
 
             if autosum:
-                out += ['.. autosummary::', '   :toctree:', '']
+                # GAEL: Toctree commented out below because it creates
```

Comment:
sorry it was in a comment above that got shrunk by GitHub -- basically these files are copied and modified as little as possible to match mne-python and upstream of that scikit-learn. Eventually there will be a module for building example docs since so many groups use this code, but for now we copy-paste.